static/include/snippet/show-secret.md

```diff
@@ -0,0 +1,9 @@
+{{< alert title="Location secret" color="info" >}}
+By default, the sample code does not include your robot location secret.
+We strongly recommend that you add your location secret as an environment variable and import this variable into your development environment as needed.
+
+To show your robot's location secret in the sample code, toggle **Include secret** on the **Code sample** tab.
+You can also see your location secret on the [locations page](/manage/fleet/locations/).
+{{< /alert >}}
+
+{{% snippet "secret-share.md" %}}
```

Comment:
Nesting snippets because:

- There are some places where we only want to warn of the dangers of sharing, but
- Everywhere else we want to do both:
	- warn of dangers of sharing and also:
	- advise on how to find and show location secrets.

So I include the other snippet in this one: meaning we always get the warning when we show users how to access their secret, but are still able to provide just the warning via `{{% snippet "secret-share.md" %}}` directly as needed (which we do in a few places, like when talking about parts secrets or when not discussing the **Code sample** tab).